<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGraphDialog</class>
 <widget class="QDialog" name="CreateGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>311</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>frequency (hertz)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="cbFrequency">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>50.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>amplitude </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="cbAmplitude">
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>duration </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="cbDuration">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>60.000000000000000</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbTypeGraph">
      <item>
       <property name="text">
        <string>linegraph</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>scatterplot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>histogram</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bargraph</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbTypeSensor">
      <item>
       <property name="text">
        <string>sensor 1 - electro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sensor 2 -</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sensor 3 -</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>sensor 4 -</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbColor">
      <item>
       <property name="text">
        <string>blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yellow</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnCreateGraph">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>create graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbFrequency</tabstop>
  <tabstop>cbAmplitude</tabstop>
  <tabstop>cbDuration</tabstop>
  <tabstop>cbTypeGraph</tabstop>
  <tabstop>cbTypeSensor</tabstop>
  <tabstop>cbColor</tabstop>
  <tabstop>btnCreateGraph</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
